std/logger/filelogger.d

```diff
@@ -270,3 +271,12 @@ class FileLogger : Logger
     assert(tl !is null);
     stdThreadLocalLog.logLevel = LogLevel.all;
 }
+
+@safe unittest
+{
+    // we don't need to actually run the code, only make sure
+    // it compiles
+    static _() {
+        auto l = new shared FileLogger("");
```

Comment:
what exactly does this test if we assign this to a new variable that should always work anyway?

Reply:
This is @atilaneves ' line, but, as I understand it, it tests that the templated constructor works as a shared constructor.